exercises/practice/knapsack/.meta/example.8th

```diff
@@ -0,0 +1,17 @@
+: maximum-values \ n -- a
+  n:1+ [0] swap a:smear
```

Comment:
```suggestion
  n:1+ a:new 0 a:push swap a:smear
```
Also, this doesn't appear to be "maximum-values", it appears to be an array of n+1 zeroes